lib/src/main/java/com/logentries/logger/AsyncLoggingWorker.java

```diff
@@ -342,8 +342,11 @@ public void run() {
                             }
 
                             if (message != null) {
-                                this.leClient.write(Utils.formatMessage(message.replace("\n", LINE_SEP_REPLACER),
-                                        logHostName, useHttpPost));
+                                if(sendRawLogMessage){
```

Comment:
Unless i'm missing something, the variable name seems to contradict the intent. If sendRowLogMessage is true, the raw log message along with other information (TraceID, DeviceID, Timestamp, etc...) will be written to the output. Where as if sendRowLogMessage is false, it just sends the message without any additional meta-data.